### DISCUSSION TITLE
Consider adding network to account modal

### DISCUSSION THREAD

**Discussion by frolic** [rainbow-me/rainbowkit#363] [Feedback]
Before I execute a transaction, I'll first check if the user is on the right chain and, if not, switch to the right one. With this behavior, there are some cases where I don't necessarily need to show/enable the chain switch UI. However, disabling it means that the balance in the account modal is a little unclear. I feel like there's plenty of room in there to display the current chain near the balance (and also another spot for a chain switch button/link if it felt necessary). Thoughts?

**Comment by nicoglennon** [comment]
thanks for the feedback @holic, running this by the team -- i agree chain context in the account modal would be really nice. will get back to you with our thoughts soon 🤠
